### PR TITLE
feat: add eword cli package

### PR DESCRIPTION
**Pull request by subekti404dev**
add cli package, based on #5  :pray: 
How to use:
```bash
eword pull --spreadsheet-id <google-sheet-id> --out <output-filename>
```

#hacktoberfest

### CODE REVIEW

**Comment by afrianjunior** [comment]
I think better the package name is `cli`, wdyt?

**Comment by subekti404dev** [comment]
oke updated mas @afrianjunior :pray:

File: packages/cli/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cli",
```
Comment (afrianjunior) [inline packages/cli/package.json]: sorry late mentioning this, I think better the package name is `@efishery/eword-cli`, but the folder name is still `cli`

**Review by afrianjunior — APPROVED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (subekti404dev) [reply packages/cli/package.json]: oke mas @afrianjunior , updated deui hehe :+1:

File: packages/cli/src/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
```
Comment (r17x) [inline packages/cli/src/index.js]: Bisa provided test untuk fungsi utamanya ? 

@subekti404dev

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by r17x** [comment]
@afrianjunior default branch repository ini perlu di-set menjadi `main` dan @subekti404dev  harus ganti target branch ke `main` juga setelah `default branch` sudah diatur.

**Comment by r17x** [comment]
@subekti404dev  kwkwkw jangan diset ke main kalau belom diatur sama @afrianjunior  (kena auto closed juga yah)

**Comment by subekti404dev** [comment]
eh wkwkw ... cara balikinnya gimana mang @r17x ?

**Comment by afrianjunior** [comment]
PR lagi aja, ga bisa reopen

**Pull request closed without merging**